connectors/src/connectors/confluence/index.ts

```diff
@@ -235,40 +235,17 @@ export async function cleanupConfluenceConnector(
     return new Err(new Error("Connector not found"));
   }
 
-  return sequelizeConnection.transaction(async (transaction) => {
-    await Promise.all([
-      ConfluenceConfiguration.destroy({
-        where: {
-          connectorId: connector.id,
-        },
-        transaction,
-      }),
-      ConfluenceSpace.destroy({
-        where: {
-          connectorId: connector.id,
-        },
-        transaction,
-      }),
-      ConfluencePage.destroy({
-        where: {
-          connectorId: connector.id,
-        },
-        transaction,
-      }),
-    ]);
-
-    const nangoRes = await nangoDeleteConnection(
-      connector.connectionId,
-      getRequiredNangoConfluenceConnectorId()
-    );
-    if (nangoRes.isErr()) {
-      throw nangoRes.error;
-    }
+  await connector.delete();
```

Comment:
Can this fail? Compared to github error handling we probably want to align the two?

Reply:
I thiiiiink connector deletion should come after nango deletion?

Reply:
We always have different error handling in all the cleanup functions, let me take a stab at those and align them on the GitHub example.

Reply:
Fix in https://github.com/dust-tt/dust/pull/3812/commits/620a58d5ceaf66f2b7e1a7f0738e0dbbed5280ee.